backend/src/data/repositories/courses-to-mentors/courses-to-mentors.repository.ts

```diff
@@ -25,14 +25,15 @@ class CoursesToMentors {
   public async getByUserIdAndCourseId({
```

Comment:
As for now this method won't return any record - so maybe it makes sense to name it properly